zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2579,7 +2579,7 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
-  /decision-instances/{decisionInstanceId}:
+  /decision-instances/{decisionEvaluationInstanceKey}:
```

Comment:
💭 could you clarify this with @aleksander-dytko if not already done?

Reply:
what should be clarified ?